src/stdlib_stats_distribution_exponential.fypp

```diff
@@ -0,0 +1,322 @@
+#:include "common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+module stdlib_stats_distribution_exponential
+    use stdlib_kinds, only : sp, dp, xdp, qp, int32
+    use stdlib_error, only : error_stop
+    use stdlib_random, only : dist_rand
+    use stdlib_stats_distribution_uniform, only : uni=>rvs_uniform
+
+    implicit none
+    private
+
+    real(dp), parameter  :: ONE = 1.0_dp
```

Comment:
As this is only used in a single subroutine, I suggest making it a local parameter inside that.